package.json

```diff
@@ -27,7 +29,8 @@
     "lint-fix": "eslint . --fix",
     "test": "cross-env NODE_ENV=test ava",
     "coverage": "cross-env NODE_ENV=test nyc ava",
-    "report-coverage": "nyc report --reporter=text-lcov | coveralls"
+    "report-coverage": "nyc report --reporter=text-lcov | coveralls",
+    "version": "babel-node build/version.js"
```

Comment:
Huh, I'm surprised that's not deployed yet (I guess only on a new npm release?).

The `npm version` [docs](https://docs.npmjs.com/cli/version) go into more detail and seem to be correct:

> 4\. Run the version script. These scripts have access to the new version in package.json (so they can incorporate it into file headers in generated files for example). Again, scripts should explicitly add generated files to the commit using git add.